restapi.go

```diff
@@ -224,7 +225,8 @@ func (self *RestAPI) pushNotification(reqId string, kv map[string]string, perdp
 	notif := NewEmptyNotification()
 
 	for k, v := range kv {
-		if len(v) <= 0 {
+		v = strings.Trim(v, " \n")
+		if v == "" {
```

Comment:
on len vs == "", it's a matter of personal preference... http://stackoverflow.com/questions/18594330/the-best-way-to-test-for-an-empty-string-in-go